cluster/config-defaults.yaml

```diff
@@ -458,6 +458,8 @@ dns_cache: "dnsmasq"
 dns_cache: "unbound"
 {{ end }}
 
+dns_unbound_liveness_probe: "true"
```

Comment:
Mark it as `experimental_`? Not sure why we'd want to disable it, if it works.

Reply:
Or just make it unconditional.

Reply:
Added `experimental_`. I just want to be able to disable it in case something didn't work as expected after we roll it out to different channels. At least for a while.